docs/changelog.md

```diff
@@ -2,6 +2,19 @@
 
 ## CARET
 
+### v0.5.1 <small>\_ Mar 11, 2024</small> {id = "0.5.1"}
+
+- **Update** Remove the description of response time histogram. ([caret_doc #331](https://github.com/tier4/caret_doc/pull/331))
```

Comment:
```suggestion
- **Update**:  Remove the description of response time histogram. ([caret_doc #331](https://github.com/tier4/caret_doc/pull/331))
```
Insert colons in other places as well.

Reply:
Fixed after `v0.4.22` in https://github.com/tier4/caret_doc/pull/334/commits/d002e71f9981ec5c1e8aa3c208c3f2cee5f822a1 .